recipes/nv-codec-headers/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,8 @@
+/* example was taken from https://www.gnu.org/ghm/2011/paris/slides/andreas-enge-mpc.pdf */
+
+#include <ffnvcodec/nvEncodeAPI.h>
+#include <iostream>
+
+int main () {
+    std::cout << "hello NVENC API version " << NVENCAPI_MAJOR_VERSION << "." << NVENCAPI_MINOR_VERSION << "\n";
+}
```

Comment:
We tend to use c examples as much as possible.
Please rename this file to `test_package.c` + modify `test_package/CMakeLists.txt`.
This tests whether nv-codec-header is really a c header and isn't dependent on any c++ standard library.

```suggestion
/* example was taken from https://www.gnu.org/ghm/2011/paris/slides/andreas-enge-mpc.pdf */

#include "ffnvcodec/nvEncodeAPI.h"
#include <stdio.h>

int main () {
    printf("hello NVENC API version %d.%d\n", NVENCAPI_MAJOR_VERSION, NVENCAPI_MINOR_VERSION);
}
```